task_15/task_7.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+let result;
+
+let numbers = prompt();
+let sum = 0;
+console.log(numbers);
+
+let i = 0;
+while (i < numbers.length) {
+    if(i == 0) {
```

Comment:
If you set initial value `let sum = 1` you will remove this `if`

Reply:
correct